packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -2098,4 +2100,919 @@ describe("MultiSelect", () => {
+                            onValidate={onValidate}
+                            required={requiredMessage}
+                            selectedValues={["1"]}
+                        />,
+                    );
+
+                    // Assert
+                    expect(onValidate).toHaveBeenCalledExactlyOnceWith(null);
+                });
+            });
+
+            describe("picking a value after there was an error", () => {
+                it("should still be in an error state before a value is picked", async () => {
+                    // Arrange
+                    const requiredMessage = "Required field";
+                    const {userEvent} = doRender(
+                        <ControlledMultiSelect required={requiredMessage} />,
+                    );
+                    await userEvent.tab();
+                    await userEvent.tab(); // Tab through the select to trigger error
+
+                    // Act
+                    await userEvent.click(await screen.findByRole("button")); // Open the dropdown
+
+                    // Assert
+                    expect(await screen.findByRole("button")).toHaveAttribute(
+                        "aria-invalid",
+                        "true",
+                    );
+                });
+
+                it("should not be in an error state once a value is picked", async () => {
+                    // Arrange
+                    const requiredMessage = "Required field";
+                    const {userEvent} = doRender(
+                        <ControlledMultiSelect required={requiredMessage} />,
+                    );
+                    await userEvent.tab();
+                    await userEvent.tab(); // Tab through the select to trigger error
+                    await userEvent.click(await screen.findByRole("button")); // Open the dropdown
+
+                    // Act
+                    await userEvent.click(await screen.findByText("item 1")); // Pick a value
+
+                    // Assert
+                    expect(await screen.findByRole("button")).toHaveAttribute(
+                        "aria-invalid",
+                        "false",
+                    );
+                });
+            });
+
+            it("should use the default required error message if required is set to true", async () => {
+                // Arrange
+                const onValidate = jest.fn();
+                const {userEvent} = doRender(
+                    <ControlledMultiSelect
+                        onValidate={onValidate}
+                        required={true}
+                    />,
+                );
+
+                // Act
+                await userEvent.tab();
+                await userEvent.tab();
+
+                // Assert
+                expect(onValidate).toHaveBeenCalledExactlyOnceWith(
+                    "This field is required.",
+                );
+            });
+        });
+
+        describe("validate and required props", () => {
```

Comment:
SingleSelect doesn't have these tests since we don't have a way to clear/unselect the selected value